### DISCUSSION TITLE
Pick Multiple Images

### DISCUSSION THREAD

**Discussion by phuvinhng** [react-native-image-picker/react-native-image-picker#1584] [Q&A] [answered]
Hi guys,

I have the questions about the '**launchImageLibrary**', 
Seem I only can pick **one images in once**, 
So I don't know the library '**react-native-image-picker**' **only support pick one image in once** or I **have to change the logic in my mind to pick up more images**?

Thanks.

**Comment by whirlxd** [comment]
i have the same question , @phuvinhng did you find any solution

**Comment by Johan-dutoit** [comment, marked as answer]
The docs have a property called `selectionLimit`.
